action.js

```diff
@@ -54,6 +54,25 @@ async function addComment(client, taskId, commentId, text, isPinned) {
   }
 }
 
+async function updateCustomField(client, taskId, fieldName, content) {
+  const task = await client.tasks.findById(taskId);
+  const targetCustomField = task.custom_fields.find(
+    (field) => field.name === fieldName
+  );
+  if (!targetCustomField) {
+    core.info(`The custom field "${fieldName}" does not exist in project`);
+    return;
+  }
+  try {
+    await client.tasks.update(taskId, {
+      custom_fields: { [targetCustomField.gid]: content },
+    });
+    core.info(`Custom fields ${fieldName} updated to: ${content}`);
+  } catch (error) {
+    console.error("rejecting promise", error);
```

Comment:
just wondering if this should return an error rather than just "console.error". Does the workflow fail in this case? Let's make it consistent with the other functions